### ISSUE TITLE
Documents parameters you can query for occurrence download

### ISSUE THREAD

**Issue by ManonGros** [gbif/tech-docs#58]
I would have expected to find those here: https://tech-docs.gbif-uat.org/en/data-use/api-downloads#requesting-an-occurrence-download

I am told they are the same as for the occurrence search: https://tech-docs.gbif-uat.org/en/openapi/v1/occurrence#/Searching%20occurrences/searchOccurrence

**Comment by CecSve** [comment]
@ManonGros is it this section where you want the information from the links currently in the text included: https://techdocs.gbif-uat.org/en/data-use/api-downloads#searchparameters?

**Comment by ManonGros** [comment]
Yes but to be honest, it would probably be enough to retitle the paragraph "Occurrence search and download parameters". Just to make it clear that they are the same.
